### ISSUE TITLE
Readme producerStream example doesn't work

### ISSUE THREAD

**Issue by fmsbeekmans** [ovotech/fs2-kafka-client#4]
Looks like the readme has become outdated. produceRecord is called with 2 parameter lists in the stream example but not in the first produce example.

```
    ...
    Stream.eval{
      produceRecord[IO](producer)(
        new ProducerRecord[String, String](topic,key, value))
    }
    ...
```

should be 
```
    ...
    Stream.eval{
      produceRecord[IO](
        producer,
        new ProducerRecord[String, String](topic,key, value)
      )
    }
    ...
```

**Comment by filosganga** [comment]
This has been caused by the #11.

**Comment by filosganga** [comment]
This has been addressed via #19
